bazel/protobuf.patch

```diff
@@ -64,6 +72,7 @@ index 32b26cbdc..e28b8e387 100644
 +    actual = select({
 +        ":linux-aarch_64": "@com_google_protobuf_protoc_linux_aarch_64//:protoc",
 +        ":linux-x86_64": "@com_google_protobuf_protoc_linux_x86_64//:protoc",
++        ":linux-ppcle_64": "@com_google_protobuf_protoc_linux_ppcle_64//:protoc",
```

Comment:
upstream patch?

Reply:
This is one patch file that existed before my changes (I just added the ppclc_64 case). I have not created an upstream patch for this one either.

Reply:
this is our patch to try and prevent building of protoc